doc-source/configuration.rst

```diff
@@ -110,10 +121,16 @@ As a minimum, the table MUST contain the keys ``name`` and ``version``.
 		[project]
 		readme = "README.rst"
 
-		[project.readme]
-		file = "README.rst"
-		content-type = "text/x-rst"
-		encoding = "UTF-8"
+	.. code-block:: TOML
+
+		[project]
+		readme = {
+			file = "README.md" ,
+			content-type = "text/markdown",
+			encoding = "UTF-8",
+		}
```

Comment:
This is not valid TOML syntax. Inline tables _must_ be a single line, and must not have a trailing comma. I find it _really_ irritating - "users", not "authors" write TOML and get tripped up by this. It's allowed on arrays, but not tables.